lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -301,17 +301,11 @@ html.video-fullscreen {
           &.fa-arrow-circle-right {
             top: 30%;
           }
-        }
-        &:last-child {
-          > .fa {
+          &.exam-without-proctoring {
```

Comment:
Is this actually all exams that are not proctored, timed and otherwise?  Are these timed exams?  Are these proctored exams where the learner opted out of proctoring?
If it is the latter, maybe?:
```
 &.exam-with-proctoring-optout {
```
Also, can you help me understand how this is used without a reference somewhere else?  Thanks.